api/controllers/shabads.js

```diff
@@ -261,12 +261,13 @@ exports.shabads = async (req, res) => {
 };
 
 exports.angs = async (req, res) => {
-  let PageNo = parseInt(req.params.PageNo, 10);
+  let { PageNo } = req.params;
   const sinceDate = req.query.updatedsince ? lib.isValidDatetime(req.query.updatedsince) : null;
 
-  if (PageNo < 1) {
+  if (!lib.isRangeOfNumbers) {
```

Comment:
Wait where are we passing the args for this function call?

Reply:
think this should be `if (!lib.isRangeOfNumber(PageNo))`